### ISSUE TITLE
Array wildcard validation is broken

### ISSUE THREAD

**Issue by vladshcherbin** [mikeerickson/validatorjs#205]
From the docs, this example:

```js
const data = {
  users: [
    {
      name: 'John',
      bio: {
        age: 28,
        education: {
          primary: 'Elementary School',
          secondary: 'Secondary School'
        }
      }
    }
  ]
}

const rules = {
  'users.*.name': 'required',
  'users.*.bio.age': 'min:18',
  'users.*.bio.education.primary': 'string',
  'users.*.bio.education.secondary': 'string'
}

const validation = new Validator(data, rules)

if (validation.fails()) {
  console.log(validation.errors.all())
}
```

It gives error:

![screen shot 2017-07-21 at 10 50 08 am](https://user-images.githubusercontent.com/6711845/28454063-799f6258-6e02-11e7-9cd3-fa1d256b8439.png)

The field is actually defined, but the error is given and error key is a string without array index.

Package version - 3.13.3

**Comment by alanhr** [comment]
Hi @VladShcherbin , the wildcard validation has not been added yet. :/

**Comment by vladshcherbin** [comment]
@alanhr yes, I found out that although it was added more than a month ago, a new version is still not published. I linked this issue to https://github.com/skaterdav85/validatorjs/issues/204, docs are misleading now.

Thank you, hope a new version will be released soon.

**Comment by mikeerickson** [comment]
@VladShcherbin I am aiming to have a new release by EOD Wednesday @alanhr

**Comment by vladshcherbin** [comment]
@mikeerickson hey, any news on a new release?

**Comment by vladshcherbin** [comment]
Fixed in 3.13.4.
